Feliz.Recharts/Tooltip.fs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Feliz.Recharts
+
+open System
+open Feliz
+open Fable.Core
+open Fable.Core.JsInterop
+
+
+[<Erase>]
+type tooltip =
+    static member inline separator (value: string) = Interop.mkAttr "separator" value
+    static member inline offset (value: int) = Interop.mkAttr "offset" value
+    static member inline filterNull (value: bool) = Interop.mkAttr "filterNull" value
+    static member inline cursor (value: bool) = Interop.mkAttr "cursor" value
+    static member inline cursor(?stroke: string, ?strokeWidth: int) =
+        let cursor = createObj [
+            "stroke" ==> Option.defaultValue "" stroke
+            "strokeWidth" ==> Option.defaultValue 0 strokeWidth
+        ]
+
+        Interop.mkAttr "cursor" cursor
+    static member inline active (value: bool) = Interop.mkAttr "active" value
+    static member inline content (render: ITooltipProperty list -> ReactElement) = Interop.mkAttr "content" render
```

Comment:
`ITooltipProperty list` does not seem right, the [docs](https://recharts.org/en-US/examples/CustomContentOfTooltip) show that the parameter is an object `({ active, payload, label }) =>`.

Reply:
> so what would be the best way of fixing this?

@kerams I haven't been keeping up with changes in recharts but if you say the parameter object has type `({ active, payload, label })` then using an anonymous record should work